WMFComponents/Sources/WMFComponents/Components/Activity Tab/WMFActivityTabView.swift

```diff
@@ -229,38 +231,37 @@ public struct WMFActivityTabView: View {
                     }
                     .frame(width: 38, height: 38)
                     .clipShape(Circle())
-                    .overlay(Circle().stroke(Color.white, lineWidth: 2))
                 }
 
                 let remaining = totalSavedCount - 3
                 Text(viewModel.localizedStrings.remaining(remaining))
-                    .font(.system(size: 14, weight: .medium))
+                    .font(Font(WMFFont.for(.caption2)))
                     .foregroundColor(.white)
                     .frame(width: 38, height: 38)
-                    .background(Circle().fill(Color.gray))
-                    .overlay(Circle().stroke(Color.white, lineWidth: 2))
             }
         }
     }
 
     private func articlesReadGraph(weeklyReads: [Int]) -> some View {
-        Chart {
-            ForEach(weeklyReads.indices, id: \.self) { index in
-                BarMark(
-                    x: .value(viewModel.localizedStrings.week, index),
-                    y: .value(viewModel.localizedStrings.articlesRead, weeklyReads[index] + 1),
-                    width: 12
-                )
-                .foregroundStyle(weeklyReads[index] > 0 ? Color(uiColor: theme.accent) : Color(uiColor: theme.baseBackground))
-                .cornerRadius(1.5)
+        Group {
+            Chart {
+                ForEach(weeklyReads.indices, id: \.self) { index in
+                    BarMark(
+                        x: .value(viewModel.localizedStrings.week, index),
+                        y: .value(viewModel.localizedStrings.articlesRead, weeklyReads[index] + 1),
+                        width: 12
+                    )
+                    .foregroundStyle(weeklyReads[index] > 0 ? Color(uiColor: theme.accent) : Color(uiColor: theme.border))
```

Comment:
Maybe try `Color(uiColor: theme.newBorder)` instead here - I see that .light for that uses `gray300`, which is what the Figma shows.